src/types/abstract_hmm.jl

```diff
@@ -109,9 +112,16 @@ function obs_distributions end
 
 ## Fallbacks for no control
 
+initialization(hmm::AbstractHMM, ::Nothing) = initialization(hmm)
 transition_matrix(hmm::AbstractHMM, ::Nothing) = transition_matrix(hmm)
-log_transition_matrix(hmm::AbstractHMM, ::Nothing) = log_transition_matrix(hmm)
+log_transition_matrix(hmm::AbstractHMM, ::Nothing) = log_transition_matrix(hmm) # Is this function needed? If `log_transition_matrix(hmm, nothing)`, then `transition_matrix(hmm, nothing)` returns `transition_matrix(hmm)`.
```

Comment:
If I remember correctly, it is useful to avoid computing the elementwise log and instead choosing the efficient code path when it exists

Reply:
Cool! I didn't knew that... but it makes sense: it would mean adding less calls to the stack. Thanks! I will delete the comment.